js/load.js

```diff
@@ -41,6 +42,7 @@ Vue.component('PkpButton', PkpButton);
 Vue.component('Spinner', Spinner);
 Vue.component('Tab', Tab);
 Vue.component('Tabs', Tabs);
+Vue.component('FieldMappedAutosuggest', FieldMappedAutosuggest);
```

Comment:
I didn't want to refactor the whole interface to use the `new Form()`, so I just made this component accessible.